nodejs/week3/calculator/index.js

```diff
@@ -0,0 +1,48 @@
+const express = require("express");
+const app = express();
+
+app.use(express.urlencoded({ extended: true }));
+// Parse JSON bodies (as sent by API clients)
+app.use(express.json());
+
+//////
+app.get("/calculator/add", async (request, response) => {
+  let result = Number(request.query.first) + Number(request.query.second);
+  response.send(`the result is ${result}`);
+});
+app.get("/calculator/multiply", async (request, response) => {
+  let result = Number(request.query.first) * Number(request.query.second);
+  response.send(`the result is ${result}`);
+});
+app.get("/calculator/divide", async (request, response) => {
+  let result = Number(request.query.first) / Number(request.query.second);
+  response.send(`the result is ${result}`);
+});
+app.get("/calculator/subtract", async (request, response) => {
+  let result = Number(request.query.first) - Number(request.query.second);
+  response.send(`the result is ${result}`);
+});
+/////////////
+app.post("/calculator/add", async (request, response) => {
```

Comment:
Good job 👍 
You can use array.reduce method if you wanted to take it further - as what happens if the user wants to work with more numbers, and we are not sure how many numbers there is going to be.